### ISSUE TITLE
Missing `initialize` from `vscode/extensions`

### ISSUE THREAD

**Issue by char0n** [CodinGame/monaco-vscode-api#283]
Hi @CGNonofr,

It seems that following import now longer works:

```js
import { initialize } from 'vscode/extensions'
```

`initialize` is no longer part of that file. Affected version is `@codingame/monaco-vscode-api@=1.83.15`.

**Comment by char0n** [comment]
This documentation https://github.com/CodinGame/monaco-vscode-api#vscode-api-usage doesn't seem to be aligned with the actual code.

**Comment by char0n** [comment]
I figured I need to use:

```js
import 'vscode/localExtensionHost';`
```

But it seems, that things will starts working only after I perform some action (like opening a command palette):

https://github.com/CodinGame/monaco-vscode-api/assets/193286/6317e9ff-0c93-427a-b7d3-47df01b39a23

**Comment by char0n** [comment]
I did dig deeper and the issue is with monaco service initialization. The simplest code to reproduce:

```js
import {  initialize as initializeMonacoServices } from 'vscode/services';
import 'vscode/localExtensionHost';

console.dir('initializing');
await initializeMonacoServices({});
console.dir('initialized');
```

`initialized` is never printed and the promise returned from `initializeMonacoServices` never resolves. Any idea what would be the problem?

**Comment by CGNonofr** [comment]
Working on way to improve the changelog

> This documentation https://github.com/CodinGame/monaco-vscode-api#vscode-api-usage doesn't seem to be aligned with the actual code.

Right, will fix

> initialized is never printed and the promise returned from initializeMonacoServices never resolves. Any idea what would be the problem?

No idea, I'm interested if you find the cause of it

**Comment by CGNonofr** [comment]
It'll be fixed soon in #285

in the meantime, you can add this workaround:
```typescript
import { IExtensionService } from 'vscode/vscode/vs/workbench/services/extensions/common/extensions'
import { registerServiceInitializeParticipant } from 'vscode/lifecycle'

registerServiceInitializeParticipant(accessor => accessor.get(IExtensionService))
```

**Comment by char0n** [comment]
> ```ts
> import { IExtensionService } from 'vscode/vscode/vs/workbench/services/extensions/common/extensions'
> import { registerServiceInitializeParticipant } from 'vscode/lifecycle'
> 
> registerServiceInitializeParticipant(accessor => accessor.get(IExtensionService))
> ```

I can confirm, the workaround works. Thanks!
